src/reducers.js

```diff
@@ -167,8 +168,20 @@ export const resetWasWatered = plotContent =>
  * @param {?farmhand.plotContent} plotContent
  * @returns {?farmhand.plotContent}
  */
-export const resetWasShoveled = plotContent =>
-  plotContent && plotContent.wasShoveledToday ? null : plotContent
+export const resetWasShoveled = plotContent => {
```

Comment:
i updated the `wasShoveledToday` to be a bit more complex to allow for shoveled plots to stay in the shoveled state for multiple days. also would like your feedback on how this was done @jeremyckahn 😄

Reply:
I think this is a good addition to the game! Since there's now a truthy `daysUntilClear` value, I wonder if we need the `wasShoveled` boolean anymore. Could that be removed now?

If I'm reading this right, then `plotContent`s could have `daysUntilClear` of `0` but still not be cleared (they would be cleared the next time this is run). To me it seems that `plotContent.daysUntilClear === 0` should mean that the plot should be cleared _now_. What do you think @lstebner?

Reply:
i overlooked this comment for some reason, but yeah what you said is true. these don't really "clear" until the value falls below 0. i can update so it clears at 0, but i will probably also then go +1 the days until clear value when shoveling since i've been testing with this code in place and it's felt pretty good.

Reply:
made this change!